mpire/comms.py

```diff
@@ -159,43 +167,45 @@ def task_completed_progress_bar(self, progress_bar_last_updated: Optional[dateti
 
         # Check if we need to update
         now = datetime.now()
-        if force_update or (now - progress_bar_last_updated).total_seconds() > 0.1:
-            self._task_completed_queue.put(progress_bar_n_tasks_completed)
+        if force_update or (now - progress_bar_last_updated).total_seconds() > self.progress_bar_update_interval:
```

Comment:
Looking at the function arguments, `progress_bar_last_updated` can be None. This will crash when you will pass None (although I suspect you will always pass some datetime here)

Reply:
Good catch. The typing information seems to be wrong, I'm always passing a datetime object, indeed. Will update it and merge.